bindings/java/src/org/sleuthkit/datamodel/CaseDbAccessManager.java

```diff
@@ -675,5 +723,226 @@ private void validateSQL(String sql) throws TskCoreException {
 		 * TODO (JIRA-5950): Need SQL injection defense in CaseDbAccessManager 
 		 */
 	}
+	
+	/**
+	 * A wrapper around a PreparedStatement to execute queries against the
+	 * database.
+	 */
+	@Beta
+	public static class CasePreparedStatement implements AutoCloseable {
```

Comment:
For consistency with CaseDbAccessManager and CaseDbConnection, and perhaps for clarity, I would rename this class as CaseDbPreparedStatement.